src/core_plugins/kibana/common/tutorials/onprem_cloud_instructions.js

```diff
@@ -17,25 +17,41 @@
  * under the License.
  */
 
-export const TRYCLOUD_OPTION1 = {
-  title: 'Option 1: Try module in Elastic Cloud',
-  textPre: 'Go to [Elastic Cloud](https://www.elastic.co/cloud/as-a-service/signup?blade=kib). Register if you ' +
-           'do not already have an account. Free 14-day trial available.\n\n' +
-
-           'Log into the Elastic Cloud console\n\n' +
-
-           'To create a cluster, in Elastic Cloud console:\n' +
-           ' 1. Select **Create Deployment** and specify the **Deployment Name**\n' +
-           ' 2. Modify the other deployment options as needed (or not, the defaults are great to get started)\n' +
-           ' 3. Click **Create Deployment**\n' +
-           ' 4. Wait until deployment creation completes\n' +
-           ' 5. Go to the new Cloud Kibana instance and follow the Kibana Home instructions'
+import { i18n } from '@kbn/i18n';
 
+export const TRYCLOUD_OPTION1 = {
+  title: i18n.translate('kbn.common.tutorials.premCloudInstructions.option1.title', {
+    defaultMessage: 'Option 1: Try module in Elastic Cloud',
+  }),
+  textPre: i18n.translate('kbn.common.tutorials.premCloudInstructions.option1.textPre', {
+    // eslint-disable-next-line
+    defaultMessage: 'Go to [Elastic Cloud]({link}). Register if you \
+      do not already have an account. Free 14-day trial available.\n\n\
+      Log into the Elastic Cloud console\n\n\
+      To create a cluster, in Elastic Cloud console:\n\
+      1. Select **Create Deployment** and specify the **Deployment Name**\n\
```

Comment:
I don't think it will make a difference in the output text, but let's keep the extra space of indentation in front of each number here to keep a clear outline